### ISSUE TITLE
Meeting Info

### ISSUE THREAD

**Issue by NanaRei** [aeimskei/cyber-security-club#18]
Can someone please change the meeting info to:

1st club meet up: Tuesday, August 28th, 3 - 6 pm

Welcome to the Cyber Security Club! This is our first networking event where you will be introduced to club leaders, our mission for this semester, and how you can be involved. We will have a presentation, followed by a question/answer session, and networking at the end. For a complete schedule: 

3:00 - 3:30 pm    Networking
3:30 - 4:30 pm    Presentation (includes activities)
4:30 - 5:00 pm    Question/Answer Session 
5:00 - 5:30 pm    Networking 

Feel free to invite your friends to attend our first meet up.

**Comment by NanaRei** [comment]
As far a the "look" design of that block of text I leave it up to you on whether you want a pop out or have them click on something that navigates somewhere or ..... It is really up to you but I want to try and have an agenda for each meeting if possible so students know what to expect.

Should we even create an Eventbrite or Meet Up group for club meetings like these? I think it would help with analytics and numbers for when we order food. 1st meeting I am keeping it light, refreshments and appetizers.

**Comment by NanaRei** [comment]
Can you have this replace what is in the block for "Meeting Info"? Take out the Monday and Tuesday and replace it with "1st club meet up: Tuesday, August 28th, 3 - 6 pm". The rest of the text you can have a pop out. See if it fits first. The text might be too wordy for that space.

**Comment by aeimskei** [comment]
Sounds good. Will do! :)

**Comment by NanaRei** [comment]
Thanks @aeimskei.

**Comment by aeimskei** [comment]
@NanaRei where's the location?

**Comment by NanaRei** [comment]
Let's make the meeting location Science 37.
